src/analysis/retail/shaman/enhancement/modules/normalizers/MaelstromWeaponCastNormalizer.tsx

```diff
@@ -1,19 +1,36 @@
-import { AnyEvent, EventType } from 'parser/core/Events';
+import { AnyEvent, EventType, HasAbility, HasRelatedEvent } from 'parser/core/Events';
 import EventsNormalizer from 'parser/core/EventsNormalizer';
 import { MAELSTROM_WEAPON_INSTANT_CAST } from './EventLinkNormalizer';
+import { MAELSTROM_WEAPON_ELIGIBLE_SPELLS } from '../../constants';
+import { Options } from 'parser/core/Analyzer';
 
 /** This normalizer removes the begincast event, and fabricated beginchannel and endchannel
  * events for instant cast enhancement spells */
 
+const MAELSTROM_WEAPON_ELIGIBLE_SPELL_IDs = MAELSTROM_WEAPON_ELIGIBLE_SPELLS.map(
+  (spell) => spell.id,
+);
+
 class MaelstromWeaponCastNormalizer extends EventsNormalizer {
+  constructor(options: Options) {
+    super(options);
+
+    this.priority = 10;
```

Comment:
Doesn't seem like this is used anywhere
```suggestion
```

Reply:
It's used to force this normalizer to run after all the others to ensure when it runs the relevant events have the required links. I suppose I could remove this and have it run at the default priority of `0` as the other normalizers have been given a negative priority but I think an explicitly defined priority avoids conflicts down the track.

Reply:
Ok sounds good, just leave a comment explaining that then